browser/model/installable-item.js

```diff
@@ -72,6 +72,10 @@ class InstallableItem {
     return this.productVersion;
   }
 
+  getProductSize() {
```

Comment:
It is up to you, but if you add new function, please, add unit test for that as well.